src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/fix/data-broker-profiles/automatic-remove/AutomaticRemoveView.tsx

```diff
@@ -91,34 +146,23 @@ export function AutomaticRemoveView(props: Props) {
         </div>
         <div className={styles.content}>
           <div className={styles.upgradeToggleWrapper}>
-            <div className={styles.upgradeToggle}>
-              <button
-                onClick={() => {
-                  setSelectedPlanIsYearly(!selectedPlanIsYearly);
-                  recordTelemetry("button", "click", {
-                    button_id: "selected_yearly_plan",
-                  });
-                }}
-                className={`${selectedPlanIsYearly ? styles.isActive : ""}`}
-              >
+            <BillingPeriodRadioMenu
+              aria-label={l10n.getString(
+                "fix-flow-data-broker-profiles-automatic-remove-features-select-plan-monthly-button",
+              )}
+              onChange={() => setSelectedPlanIsYearly(!selectedPlanIsYearly)}
```

Comment:
I believe there's often a risk of someone e.g. clicking the "yearly" button twice quickly, in which case we probably want to set it to yearly instead of toggling it twice and having it get set back to monthly.

```suggestion
              onChange={(selectedBillingPeriod: "yearly" | "monthly") => setSelectedPlanIsYearly(selectedBillingPeriod === "yearly")}
```